.github/workflows/nightly_macos_apple_silicon.yml

```diff
@@ -14,14 +14,17 @@ jobs:
   test-and-build:
     name: Rust tests, build and package nightly release
     runs-on: [self-hosted, macOS, ARM64]
+    env:
+      LIBRARY_PATH: /opt/homebrew/Cellar/zstd/1.5.6/lib
```

Comment:
Due to homebrew package updates this is now required somehow even though we already used zstd before.